gravitee-am-service/src/main/java/io/gravitee/am/service/exception/RateLimitException.java

```diff
@@ -27,6 +29,6 @@ public RateLimitException(String message) {
 
     @Override
     public int getHttpStatusCode() {
-        return 0;
+        return OK_200;
```

Comment:
Why is the 200 code here? Would not the 429 code number be better than that?

Reply:
done, didn't know about this 429 :)